### PR TITLE
chore(subscriberdb): replace print_grpc to use rpc utils function

### PR DESCRIPTION
**Pull request by uri200**
Signed-off-by: Oriol Batalla <obatalla@fb.com>

## Summary

Replace _print_grpc for the new print_grpc from rpc utils

## Test Plan

make test

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: lte/gateway/python/magma/subscriberdb/protocols/m5g_auth_servicer.py [hunk 1]
```diff
@@ -67,6 +72,10 @@ def M5GAuthenticationInformation(self, request, context):
             m5gauth_vector.xres_star = m5g_ran_auth_vectors.xres_star[16:]
             m5gauth_vector.autn = m5g_ran_auth_vectors.autn
             m5gauth_vector.kseaf = m5g_ran_auth_vectors.kseaf
+            print_grpc(
```
Comment (ardzoht) [inline lte/gateway/python/magma/subscriberdb/protocols/m5g_auth_servicer.py]: nit: we could also put all these `print_grpc` calls into a `finally` block, either way it's fine for me

**Review by ardzoht — APPROVED** [review]
lgtm, left a minor comment

**Review by pshelar — APPROVED** [review]
LGTM

File: lte/gateway/python/magma/subscriberdb/protocols/m5g_auth_servicer.py (hunk 1 above)
Comment (uri200) [reply lte/gateway/python/magma/subscriberdb/protocols/m5g_auth_servicer.py]: You are right! Didn't remember Finally was executed even after return!

will refactor that

**Pull request merged**